docs/source/hardware_components.rst

```diff
@@ -26,10 +28,13 @@ Major parts used in HackRF One:
 * `W25Q80BV 8M-bit Flash <https://www.winbond.com/resource-files/w25q80bv%20revk%2020151203.pdf>`__
 
 
+Block Diagrams
```

Comment:
Please add subheadings for each of the blog diagrams in this section. My suggestion is that you use "HackRF One R1-R8 Block Diagram" as the first subheading and "HackRF One R9 Block Diagram" as the second sub heading.